docs/pages/blog/mui-x-sep-2024-price-update.md

```diff
@@ -2,57 +2,75 @@
 
-With the launch of several new components, including [Date and Time Range Pickers](/x/react-date-pickers/date-time-range-picker/), as well as upcoming Pro versions of [Charts](/x/react-charts/) and [TreeView](/x/react-tree-view/), we're updating our pricing.
+### New pricing
 
-Moving forward, each developer requires a seat. So, for example, 50 developers will now need 50 seats.
+We will soon release the first Pro features of the [Charts](/x/react-charts/) and [Tree View](/x/react-tree-view/).
+These new features will only be available under the updated pricing: moving forward, each developer requires a license. So, for example, 50 developers will need 50 licenses.
 
-We're happy to discuss volume discounts to accommodate larger teams.
-
-### Grandfathering Current Customers
+### Grandfathering current customers
 
 We value our loyal customers and want to ensure a smooth transition: current customers will be grandfathered into their existing plans.
 
-This means you can continue using and renewing the legacy license for as long as you wish.
-However, the legacy license covers only Data Grid and Date Pickers.
+This means you can continue to renew your licenses under the legacy pricing for as long as you wish.
+However, the legacy pricing only covers the previous feature scope: Data Grid and Date Pickers.
 
 :::info
-**Access to new components:** The Pro versions of Charts and Tree View, and other future Pro components, will be available exclusively through a license key under the new Pro plan model or a Premium key (Premium users will experience no change).
+**Access to new components:** The Pro versions of Charts and Tree View, and other future Pro components, will be available exclusively through the new Pro plan model or a Premium plan (Premium customers will experience no change, all Pro components are included).
 ::::
 
-### Pricing for Small Teams
+### Pricing for small teams
+
+The price is the same for customers with 10 developers or fewer.
+Actually, we are making this pricing change to be able to keep the price affordable for small teams.
 
-The price is essentially the same for customers with 10 developers or fewer, but you still need a new license key to use the Pro versions of Charts or Tree View. If that's the case, we recommend you contact our sales team and get a discount to renew your license earlier.
+However, if you want to use the new features, you will need a new license key to use the Pro versions of Charts or Tree View.
+You can [contact](https://support.mui.com/hc/en-us/requests/new) our support team to receive a new license key.
+
+### Larger teams
+
+We're happy to discuss volume discounts to accommodate larger teams.
 
 ## Changes to the Premium plan
 
+### New Pro components
+
+We will soon release the first Pro features of the [Charts](/x/react-charts/) and [Tree View](/x/react-tree-view/).
+This feature will be available to all Premium customers.
```

Comment:
```suggestion
These features will be available to all Premium customers.
```

Reply:
Perhaps there's an opportunity to ease customers regarding the actions they need to take.

```suggestion
These features will be available to all Premium customers without any further action needed.
```